iphone/Maps/Classes/NetworkingPolicy/MWMNetworkingPolicy.h

```diff
@@ -0,0 +1,18 @@
+#include "std/function.hpp"
+
+namespace networking_policy
+{
+  using MWMNetworkingPolicyFn = function<void(BOOL solver)>;
+
+  void callPartnersApiWithNetworkingPolicy(MWMNetworkingPolicyFn const & fn);
+
+  enum class NetworkingPolicyState
+  {
+    Always,
+    Session,
+    Never
+  };
+
+  void SetNetworkingPolicyState(NetworkingPolicyState state);
```

Comment:
s/SetNetworkingPolicyState/SetStage/g
s/GetNetworkingPolicyStage/GetStage/g
As these methods already in networking_policy namespace.